tests/functional/object/mcg/test_bucket_replication_with_versioning.py

```diff
@@ -46,103 +50,312 @@ def reduce_replication_delay_setup(self, add_env_vars_to_noobaa_core_class):
             new_delay_in_miliseconds (function): A function to add env vars to the noobaa-core pod
         """
         new_delay_in_miliseconds = 60 * 1000
-        new_env_var_touples = [
+        new_env_var_tuples = [
             (constants.BUCKET_REPLICATOR_DELAY_PARAM, new_delay_in_miliseconds),
             (constants.BUCKET_LOG_REPLICATOR_DELAY_PARAM, new_delay_in_miliseconds),
         ]
-        add_env_vars_to_noobaa_core_class(new_env_var_touples)
+        add_env_vars_to_noobaa_core_class(new_env_var_tuples)
 
     @pytest.fixture()
-    def buckets_with_versioning(self, bucket_factory, mcg_obj, awscli_pod_session):
+    def make_buckets_with_versioning(self, bucket_factory, mcg_obj, awscli_pod_session):
         """
-        Prepare two buckets with versioning enabled
+        A factory that creates versioned MCG buckets
 
         Args:
             bucket_factory: Fixture for creating new buckets
             mcg_obj_session: The session-scoped MCG object
             awscli_pod_session: The session-scoped AWSCLI pod
 
         Returns:
-            Tuple of two buckets: source and target
+            function: The factory function
         """
-        # Using the OC interface allows patching a repli policy on the OBC
-        bucket_a = bucket_factory(1, "OC")[0]
-        bucket_b = bucket_factory(1, "OC")[0]
 
-        put_bucket_versioning_via_awscli(mcg_obj, awscli_pod_session, bucket_a.name)
-        put_bucket_versioning_via_awscli(mcg_obj, awscli_pod_session, bucket_b.name)
+        def _factory(amount):
+            """
+            Create versioned buckets
 
-        return bucket_a, bucket_b
+            Args:
+                amount (int): The number of buckets to create
 
-    @tier1
-    @polarion_id("OCS-6294")
+            Returns:
+                list(Bucket): The created buckets
+            """
+            # Using the OC interface allows patching a repli policy on the OBC
```

Comment:
what is a repli policy? is it a typo and should be replaces with "replication"?

Reply:
I used it as a shorthand for "replication" to stay under the in-row characters limit, but after taking a second look I can just use the correct form without passing it.

Fixed in commit https://github.com/red-hat-storage/ocs-ci/pull/11438/commits/f3ae65427265628d00b584338828ff9c9de6718c